### PR TITLE
Eyes on setting

### PR DESCRIPTION
**Pull request by liemilyg**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

===================================================
**The "Settings" endpoint was a draft and never being used(I verified that)
We now implemented this endpoint with the first kind "eyesOn"**

### CODE REVIEW

**Comment by liemilyg** [comment]
The "Settings" endpoint was a draft and never being used(I verified that)
We now implemented this endpoint with the first kind "eyesOn"

**Comment by weidongxu-microsoft** [comment]
@liemilyg Please make proper fix so that CI ModelValidation would pass.

**Comment by liemilyg** [comment]
@weidongxu-microsoft Done.
regards my changes -
The "Settings" endpoint was a draft and never being used(I verified that)
We now implemented this endpoint with the first kind "eyesOn"
Thanks/

**Review by weidongxu-microsoft — APPROVED** [review]
LGTM.

Still need ARM approval.

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json [hunk 1]
```diff
@@ -7856,19 +7959,15 @@
...
     },
     "SettingsKind": {
       "description": "Describes an Azure resource with kind.",
       "properties": {
         "kind": {
           "description": "The kind of the setting",
           "enum": [
-            "UebaSettings",
-            "ToggleSettings"
+            "EyesOn"
```
Comment (chiragg4u) [inline specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json:7970]: As per Swagger update guidance this is breaking change. Please confirm if this is not being used and share the details.

**Review by chiragg4u — CHANGES_REQUESTED** [review]
:clock1:

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json (hunk 1 above)
Comment (weidongxu-microsoft) [reply specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json:7970]: @chiragg4u 
For breaking change, here is author's explanation: https://github.com/Azure/azure-rest-api-specs/pull/9355#issuecomment-625095704

Comment (liemilyg) [reply specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json:7970]: @chiragg4u 
The "Settings" endpoint was a draft and never being used(I verified that)
We now implemented this endpoint with the first kind "eyesOn"

verified that there is no use on this endpoint for the last 60 days

Comment (liemilyg) [reply specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json:7904]: @chiragg4u fixed conflicts

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json [hunk 2]
```diff
@@ -8024,7 +8123,7 @@
       },
       "type": "object"
     },
-    "ToggleSettings": {
```
Comment (chiragg4u) [inline specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json:8027]: >ToggleSettings [](start = 5, length = 14)

Again these are breaking changes but as per the author this types are not being used; based on that I'm allowing this change.

**Review by chiragg4u — APPROVED** [review]
Signed off the breaking changes based on the authors comment that the preview API is not being used.

**Comment by weidongxu-microsoft** [comment]
@liemilyg

PR is approved. Please let me know when you want to merge.

**Comment by liemilyg** [comment]
> @liemilyg
> 
> PR is approved from all. Please let me know when you want to merge.

@weidongxu-microsoft yes, I want to merge it ASAP! :)

**Pull request merged**